src/ImageSharp/Formats/Jpeg/Components/Block8x8.cs

```diff
@@ -337,6 +337,64 @@ public nint GetLastNonZeroIndex()
             }
         }
 
+        /// <summary>
+        /// Transpose the block inplace.
+        /// </summary>
+        [MethodImpl(InliningOptions.ShortMethod)]
+        public void TransposeInplace()
```

Comment:
According to codecov we have no coverage for this? It looks correct to me but better to be safe than sorry.

Reply:
Done.